Loops_and_set/task_3.py

```diff
@@ -0,0 +1,7 @@
+new_list = []
+count = int(input("Enter count of list numbers: "))
+for i in range(count):
+    new_element = int(input("Enter numbers: "))
+    new_list.append(new_element)
+uniq_elements = sorted(set(new_list))
+print(uniq_elements)
```

Comment:
Task was check if elements is unique
![image](https://github.com/MykhailoTsyba/Homework/assets/51053483/238a05d1-0faa-4d24-8efd-ce36349ae265)